src/commands/quote.ts

```diff
@@ -90,13 +90,18 @@ async function parseQuotes(client: Client, str: string) {
     }
   )
 
+
   const parsed_emotes = await extractObjects(
     parsed_mentions,
-    /(?<=<:(.+?))\d+(?=>)/g,
-    /<:(.+?):\d+>/g,
+    /(?<=<:)(.+?):\d+>/g,
```

Comment:
![image](https://github.com/kogasacord/kogasa-bot/assets/68234036/8561ca48-c558-4cc4-be43-7806ddeadaef)
Have you tested the code?